index.bs

```diff
@@ -1229,7 +1232,7 @@ Gamepad API Integration {#gamepad-api-integration}
 
   - {{XRInputSource/gamepad}} MUST NOT be included in the array returned by {{navigator.getGamepads()}}.
   - {{XRInputSource/gamepad}}'s {{Gamepad/index}} attribute MUST be <code>0</code>.
```

Comment:
>{{XRInputSource/gamepad}}'s {{Gamepad/index}} attribute MUST be <code>0</code>.

Gah this makes me so nervous.. people are going to use this and index into inputSources.... Can we file a separate issue (and label it to be an external dependency) to figure out how we want to handle this with the Gamepad spec folks? (yes, I'm looking at you @toji)

Reply:
Re-cast #583 as a broader issue to address this topic.